hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSourceManager.java

```diff
@@ -926,7 +799,8 @@ public void join() {
    * @return list of all normal sources
    */
   public List<ReplicationSourceInterface> getSources() {
-    return new ArrayList<>(this.sources.values());
+    return this.sources.values().stream().filter(source -> source.isSourceActive())
```

Comment:
curious why this change?

Reply:
This is for the ut TestMasterReplication#testLoopedReplication. Will remove this after HBASE-24743.

Reply:
ok